packages/now-next/src/index.ts

```diff
@@ -385,11 +385,14 @@ export const build = async ({
     const pagesDir = path.join(entryPath, '.next', 'serverless', 'pages');
 
     const pages = await glob('**/*.js', pagesDir);
-    const staticPageFiles = await glob('**/*.html', pagesDir);
+    staticPages = await glob('**/*.html', pagesDir);
 
-    Object.keys(staticPageFiles).forEach((page: string) => {
+    Object.keys(staticPages).forEach((page: string) => {
       const pathname = page.replace(/\.html$/, '');
-      staticPages[path.join(entryDirectory, pathname)] = staticPageFiles[page];
+      routes.push({
+        src: path.join(entryDirectory, pathname),
```

Comment:
```suggestion
        src: `${entryDirectory.replace(/\\/g, '/')}/${pathname.replace(/\\/g, '/')}`,
```

Windows paths are not valid in routes